app/styles/modules/_avatar.scss

```diff
@@ -142,56 +142,49 @@
 }
 
 .avatar-change {
-  .gravatar-logo {
-    background: image-url('gravatar.svg') no-repeat top center;
-    background-size: contain;
-    /* This reduces the size of the gravatar logo to match the other icons */
-    font-size: 30px;
-
-    /* Mimic the style of the fontello icons */
```

Comment:
do we need to remove other fontello icons, see https://github.com/mozilla/fxa-content-server/blob/master/app/styles/fontello.css#L61